src/sources/soundsourceffmpeg.h

```diff
@@ -88,6 +88,11 @@ class SoundSourceProviderFFmpeg: public SoundSourceProvider {
         return "FFmpeg";
     }
 
+    SoundSourceProviderPriority getPriorityHint() const override {
+        // FFmpeg will only be used as the last resort
+        return SoundSourceProviderPriority::LOWEST;
```

Comment:
A single priority hint might fail. We may have a criss cross dependency. 
If a sound source has the best quality for a single file format but supports other formats in a weak way.
I am not sure if a priority hint for each file extension will be sufficient. Since it is sometimes only a extension from a container format that can carry any encoding type.